masterfabric-mobile/swift-camp
# CL - List Viper Implementation

## 📋 PR Description

This PR adds the List Module using VIPER architecture in SwiftUI to display and manage a dynamic list with item selection.

---

## ✅ Checklist


---

## 🛠 Steps to Test

1.	Navigate to the List Page from the home screen.
2.	Verify the list items are displayed correctly.
3.	Use the back button to return to the home screen.

---

## 🔗 Related Links

- Documentation: https://developer.apple.com/documentation/swiftui/list

---

### 📷 Screenshots (Optional)
| HomeView Screen    | List View |
|-------------------------------|--------------------------|
|![Simulator Screenshot - iPhone 16 Pro - 2024-12-31 at 14 02 42](https://github.com/user-attachments/assets/894076b4-570d-4fa5-a70b-95021f65ec71) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-12-31 at 14 02 44](https://github.com/user-attachments/assets/1723226f-5b6c-488a-b0ba-174189e839f8) |
